src/jenkins_job_insight/main.py

```
@@ -2344,6 +2348,150 @@ async def create_jira_bug_endpoint(
+                "launch_id": launch_id,
+            }
+
+        # Get history classifications for matched tests (concurrent queries)
+        unique_test_names = list(
+            dict.fromkeys(failure.test_name for _, failure in matched)
+        )
+        classifications = await asyncio.gather(
+            *(get_history_classification(job_id, name) for name in unique_test_names)
+        )
+        history_classifications: dict[str, str] = {
+            name: cls for name, cls in zip(unique_test_names, classifications) if cls
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)